recipes/default.rb

```diff
@@ -16,7 +16,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-chef_gem 'aws-sdk' do
+chef_gem 'aws-sdk-ec2' do
   compile_time true
   version node['aws']['aws_sdk_version'] if node.key? 'aws'
```

Comment:
This line may need to change as the `aws-sdk` gem is now versioned differently to the `aws-sdk-<service>` gems. It may be as simple as just removing this version pin, or if it's allowed (I can't see it documented anywhere), pinning it pessimistically to `~> 1.1`.

Reply:
Currently I tend to just remove the version, since
1. I can't find the `node['aws']['aws_sdk_version']` on my aws nodes anyway
1. of the simplicity of what the actual plugin does, I think, it should be very compatible with upcoming versions of the `aws-sdk-ec2` gem